README.md

```diff
@@ -23,6 +23,23 @@ _For additional improvements when fully using the ember-strict-resolver monkey p
 Ember.Registry.prototype.normalize = function (i) { return i; }
 ```
 
+## Migration
+
+Migrating away from use the _ember-resolver/classic_ can be done in piecemeal by supporting a sub-set of the old resolution formats.
+
+```js
+> _app/resolver.js_
+
+import StrictResolver from 'ember-strict-resolver';
+
+export StrictResolver.create({
+  legacyMappings: {
+    'service:camelCaseNotSupported': 'service:camel-case-not-supported'
+  }
+})
```

Comment:
I think we should *not* do the actual code here, and instead suggest folks do:

```js
import Resolver from 'ember-strict-resolver';

export default class extends Resolver {
  legacyMappings = {
    'service:camelCaseNotSupported': 'service:camel-case-not-supported'
  };

  resolve(_fullName) {
    let fullName = this.legacyMappings[fullName] || fullName;

    return super.resolve(fullName);
  }
}
```

Reply:
I think this makes a lot of sense, adding this as an example for how to migrate to ember-strict-resolver instead of adding the infrastructure to support the migration? @stefanpenner thoughts?

Reply:
@rwjblue updated to only have a docs section instead of any code.